### ISSUE TITLE
Allow emails to print to logs in dev environment

### ISSUE THREAD

**Issue by patcon** [pol-is/polis-issues#137]
Right now, emails get send out for password reset, and for data exports, and perhaps others things. For the latter, it's after some processing in the math component, which calls back to server to send email afterwards.

It would be great if email bodies/metadata could be inspected during development without trying to send a real email via Mailgun SaaS (currently the only way to send).

An alternative approach would be to try to send it via postfix, but printing to logs seems simplest.

I think this is where this would need to be added:
https://github.com/pol-is/polisServer/blob/806e77e508cd8c27a8c372502c111359fdd8b28e/email/sendEmailSesMailgun.js#L63-L73

Off the top of my head, this could be enabled in one of two ways:
1. setting `devMode` to true, and always printing to logs when in this mode, or
2. added a config like `emailServer` which could be one of `logs` or `mailgun` (or some future SaaS)

**Comment by joshsmith2** [comment]
+1! This would be super useful

**Comment by patcon** [comment]
Potential other option, perhaps in a separate docker container: https://github.com/maildev/maildev

**Comment by colinmegill** [comment]
Agree, this needs to be handled for local and non `mailgun` scenarios

**Comment by patcon** [comment]
@joshsmith2 Heads up -- Data export might not be working rn anyhow: https://github.com/pol-is/polisServer/issues/39#issuecomment-434492966

**Comment by patcon** [comment]
From gitter chat: https://gitter.im/pol-is/polisDeployment?at=5eb2cacb7975db7ebfe99d9f

@crkrenn is +1 to suggestion on using `nodemailer` and transport plugins

**Comment by colinmegill** [comment]
Data export is not working. Cc Chris
